wallet/src/Pos/Wallet/Web/State/Storage.hs

```diff
@@ -784,6 +823,17 @@ flushWalletStorage = modify flushDo
                             , _wiIsReady   = False
                             }
 
+-- | Unsafe address conversion for use in migration. This will throw an error if
+--   the address cannot be migrated.
+unsafeCIdToAddress :: WebTypes.CId WebTypes.Addr -> Address
+unsafeCIdToAddress cId = case WebTypes.cIdToAddress cId of
+    Left err -> error $ "unsafeCIdToAddress: " <> err
+    Right x  -> x
```

Comment:
This can become the `migrate` method for the `Migrate` instance for the newtype wrapper for Address, rather than a top level function.

Reply:
I left this one in because the `migrate` method has result type `WAddrId` (the newtype around `Address` that acts as the migration target for `CId Addr`), and there are several uses of this function that really do want an `Address`.

Reply:
Fair enough.